htlcswitch/switch.go

```diff
@@ -779,6 +793,10 @@ func (s *Switch) handleLocalDispatch(pkt *htlcPacket) error {
 				htlc.Amount, largestBandwidth)
 			log.Error(err)
 
+			if err := s.control.Fail(pkt.circuit.PaymentHash); err != nil {
```

Comment:
Similar comment here w.r.t this not even being a full attempt, so it doesn't warrant an update to the htlc control tower.